client/thin-replica-client/src/trs_connection.cpp

```diff
@@ -356,4 +356,24 @@ TrsConnection::Result TrsConnection::readHash(Hash* hash) {
   return Result::kFailure;
 }
 
+TrsConnection::Result TrsConnection::openStateSnapshotStream(
+    const com::vmware::concord::state_snapshot_concord::StreamSnapshotRequest& request) {
+  // TODO: Add implementation
+  return Result::kSuccess;
```

Comment:
Would you mind adding a `LOG_ERROR` message that prints "unimplemented" or "todo" or something similar - here and in all the other "todo" functions? Meaning, if the function gets called without an implementation then that must be a mistake. Alternatively, we can even add an ASSERT because it would be a programming error to call those functions.